conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollections.java

```diff
@@ -181,9 +192,15 @@ public static List<Integer> newNonNullIntegerList(Iterable<Integer> iterable) {
         return integerList;
     }
 
+    // This method modifies a list that can't handle nulls. Do not use this unless the nonNullCollections flag is set
+    public static void addAllToIntegerList(Collection<Integer> addTo, int[] elementsToAdd) {
+        addAll(addTo, () -> Arrays.stream(elementsToAdd).iterator());
```

Comment:
we could add an implementation dependency on guava for something like `addAll(addTo, Ints.asList(elementsToAdd));` -- streams can add quite a lot of overhead on their own. In practice this shouldn't really matter because we'll provide a special case implementation before this is used, but we'll need to maintain the fallback codepath either way.

Reply:
It doesn't need the optimization in that function so I will just inline the simple addition.